go/base/management/commands/go_migrate_conversations.py

```
@@ -7,6 +8,86 @@
 from go.base.utils import vumi_api_for_user
 
 
+class Migration(object):
+    """Base class for migrations."""
+    name = None
+
+    def __init__(self, dry_run):
+        self._dry_run = dry_run
+
+    @classmethod
+    def migrator_classes(cls):
+        return cls.__subclasses__()
+
+    @classmethod
+    def migrator_class(cls, name):
+        for migrator_cls in cls.migrator_classes():
+            if migrator_cls.name == name:
+                return migrator_cls
+        return None
```

Maybe raise an exception instead of returning `None` here? Saves us checking return values everywhere like we're in C.